codegenerator/cli/src/rescript_types.rs

```diff
@@ -335,6 +337,12 @@ impl RescriptTypeIdent {
             Self::String => "S.string".to_string(),
             Self::ID => "S.string".to_string(),
             Self::Bool => "S.bool".to_string(),
+            Self::Timestamp => {
+                // Don't use S.unknown, since it's not serializable to json
+                // In a nutshell, this is completely unsafe.
+                "S.json(~validate=false)->(Utils.magic: S.t<Js.Json.t> => S.t<Js.Date.t>)"
+                    .to_string()
+            }
```

Comment:
So we don't want to serialize because postgres client should manage date conversion?

Reply:
Yes, it does it for us